### PR TITLE
LIVE-5865 Add listen-to-article UI to comment article templates

### PR DESCRIPTION
**Pull request by waisingyiu**
We have been developing the listen-to-article feature for the app.  This PR adds the audio UI to the comment article template.

### Function Interface

By default, the article template does not show the new listen-to-article audio UI.  The native app has to check if the `window.showListenToArticlePlayer` function exists.  If so, this function should be invoked to display the audio UI if listen-to-article is supported for this article.

The native app should also call `window.setAudioDuration` to set the audio length on the UI.

Three functions are provided to change the state of the UI among playing, stopped and loading:
- `window.listenToArticlePlayerLoading`
- `window.listenToArticlePlayerPlaying`
- `window.listenToArticlePlayerStopped`

### Screenshots

| Light Mode | Dark Mode |
| --- | --- |
| Stopped | Stopped |
| ||
| Playing | Playing |
|||
| Loading | Loading |
|||

### CODE REVIEW

**Comment by waisingyiu** [comment]
@luke-wilkins It would be great if you could take a look and check if the screenshots look good to you.

**Review by tkgnm — APPROVED** [review]
Great work!

**Comment by luke-wilkins** [comment]
Hi @waisingyiu - looks good. Not sure if this is a native side thing, but the only thing I've noticed is the colour of 'Listen to articles' and the timer in dark mode is incorrect. Should be neutral.97.

**Comment by luke-wilkins** [comment]
Also @waisingyiu - not sure if this is native, but the colour of the audio wave in dark mode should be neutral.38

**Comment by waisingyiu** [comment]
Thank you @luke-wilkins for spotting the wrong colours!  They have been fixed.  It would be great if you can take a look!

**Comment by waisingyiu** [comment]
The listen-to-article audio control is not displayed by default and remains hidden until a Javascript function `showListenToArticlePlayer` is invoked.  So it will not have impact to article views.

File: ArticleTemplates/articleTemplateContainerComment.html [hunk 1]
```diff
@@ -31,6 +31,27 @@ <h1 class="headline selectable" id="headline">__TITLE__
...
+                    <div class="audio-player__button--loading ">
+                        <div class='pulse touchpoint__button'></div>
+                    </div>
+                    <a role="button" aria-role="button" class="audio-player__button touchpoint touchpoint--primary" id="audio-play-button-new" href="x-gu://playaudio/__AUDIO_URI__">
+                        <span class="touchpoint__button play" data-icon="&#xe04b;" aria-hidden="true"></span>
+                        <span class="touchpoint__button pause" data-icon="&#xe04D;" aria-hidden="true"></span>
+                        <span class="touchpoint__label screen-readable audio-player-readable">Play</span>
+                    </a>
+                    <div class="audio-player__info">
+                        <div class="audio-player__info__label">Listen to this podcast</div>
```
Comment (michaelclapham) [inline ArticleTemplates/articleTemplateContainerComment.html:46]: Should this be changed to "Listen to this podcast" ?

Comment (waisingyiu) [reply ArticleTemplates/articleTemplateContainerComment.html:46]: Thank you @michaelclapham for spotting this!  I will correct it in another PR.

**Pull request merged**